src/api/kube.ts

```diff
@@ -1404,6 +1404,52 @@ export class KubeHelper {
     }
   }
 
+  async getOAuthClientAuthorizations(clientName: string): Promise<string[]> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.listClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations')
+
+      if (!body.items) {
+        return []
+      }
+      const oauthClientAuthorizations = body.items as any[]
+      return oauthClientAuthorizations.filter(o => o.metadata.name && o.clientName === clientName)
+    } catch {
+      return []
+    }
+  }
+
+  async deleteOAuthClientAuthorizations(oAuthClientAuthorizations: any[]): Promise<boolean> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const options = new V1DeleteOptions()
+      oAuthClientAuthorizations.forEach(e => customObjectsApi.deleteClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations', e.metadata.name, options))
+      return true
+    } catch (e) {
+      throw this.wrapK8sClientError(e)
+    }
+  }
+
+  async consoleLinkExists(name: string): Promise<boolean> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      await customObjectsApi.getClusterCustomObject('console.openshift.io', 'v1', 'consolelinks', name)
+      return true
+    } catch {
+      return false
```

Comment:
I think `return false` only if we got 404.
Will we have any issues on k8s then?

Reply:
No, let's return false